beacon-chain/rpc/eth/beacon/validator.go

```diff
@@ -38,11 +38,11 @@ func (e *invalidValidatorIdError) Error() string {
 }
 
 // GetValidator returns a validator specified by state and id or public key along with status and balance.
-func (bs *Server) GetValidator(ctx context.Context, req *ethpb.StateValidatorRequest) (*ethpb.StateValidatorResponse, error) {
+func (beaconServer *Server) GetValidator(ctx context.Context, req *ethpb.StateValidatorRequest) (*ethpb.StateValidatorResponse, error) {
```

Comment:
Go style generally prefers smaller variable names. https://github.com/golang/go/wiki/CodeReviewComments#variable-names

In this case, I would recommend leaving as it is. It's also identical in the every other services. (If not, we should shorten it)

Reply:
So interesting... thanks for sharing that. I *personally* find that more verbose and meaningful variable names can make it easier to understand what's happening (except for iterators, indices, and the like)... Is this convention something we want to re-evaluate, or are you + team happy with the shorthand?

(meta: I see that this is used in many many places, my intent was to quietly stage this, get the build working, and then use it to start a convo about `is_optimistic` + conventions on Monday haha)

(Also, no pressure to respond to these whatsoever - super sorry if my tinkering is generating GH notifications for you)

Reply:
I agree there's a place for more verbose and meaningful variable names, but given that this is the receiver name, I'm more inclined to follow the Golang standard and strive for consistency. It's essential to follow these best practices for idiomatic Go code, and it also makes it easier to onboard other Go devs : ) 

More on how to name receiver: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

Reply:
Very cool - thank you for providing learning resources along with your feedback. I'm coming from C#/.NET so I have some norms to unlearn ^^